### PR TITLE
remove dependency on `xargs`

### PR DESCRIPTION
**Pull request by jkroepke**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

**Comment by jkroepke** [comment]
@faust64 is there a chance that you can test this branch?

**Comment by faust64** [comment]
damn, that was fast!
sorry I didn't see you there... Sure, will test now, thanks again!

File: scripts/commands/helm.sh [hunk 1]
```diff
@@ -24,19 +24,25 @@ Typical usage:
...
+        set +f # Enable globbing
+        for file in "${decrypted_file_list_dir}"/*.file; do
+            set -f # Disable globbing
+
+            if [ -e "$file" ]; then # Make sure it isn't an empty match, in case of no files
+                decrypted_file=$(cat "$file")
+                rm -- "$(printf '%s' "$decrypted_file")"
+
+                if [ "${QUIET}" = "false" ]; then
+                    printf "[helm-secrets] Removed: %s\n" "$decrypted_file"
```
Comment (faust64) [inline scripts/commands/helm.sh:40]: >&2
otherwise shows in stdout, mixed in with rendered yamls

Comment (jkroepke) [inline scripts/commands/helm.sh:40]:
```suggestion
                    printf "[helm-secrets] Removed: %s\n" "$decrypted_file" >&2
```

**Comment by faust64** [comment]
works great!
thanks!

File: scripts/commands/helm.sh (hunk 1 above)
Comment (faust64) [reply scripts/commands/helm.sh]: can't resolve that conversation myself, please do, LGTM

**Pull request merged**